docs/reference/glossary.md

```diff
@@ -442,244 +378,240 @@ A SpatialOS project includes (but isn't limited to):
 
-SpatialOS manages the world, keeping track of all the SpatialOS entities and what state they’re in. Changes to the world are made by [workers](#worker). Each worker has a view onto the world (the part of the world that they’re [interested in](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#interest)), and SpatialOS sends them updates when anything changes in that view.
+The GDK populates this world with [ECS entities](#unity-ecs-entity) that represent [SpatialOS entities](#spatialos-entity) and are in the [worker’s view](#worker-s-view). 
 
-It’s important to recognize this fundamental separation between the SpatialOS world and the view (or representation) of that world that a worker [checks out](#checking-out) locally. This is why workers must send updates to SpatialOS when they want to change the world: they don’t control the canonical state of the world, they must use SpatialOS APIs to change it.
+Additionally, the worker adds [ECS systems](#unity-ecs-system) to this world to define the logic that should be run on those ECS entities. The GDK synchronizes this world with the worker’s view stored in the [SpatialOS Runtime](#spatialos-runtime).
 
-### Write access
+## Write access
 
-> Also referred to as "authority".
+<span style="display: block;margin-top: -20px;font-size: 10pt;"><i>Also known as "authority".</i></span>
 
-Many [workers](#worker) can connect to a [SpatialOS world](#spatialos-world). To prevent them from clashing, SpatialOS only allows one worker at a time to write to each [SpatialOS component](#spatialos-component). Write access is defined at the component level.
+Many [worker instances](#worker) can connect to a [SpatialOS world](#spatialos-world). However, at most one worker instance can have write acesss on a component on a [SpatialOS entity](#spatialos-entity). This worker instance is the only one that can modify the component’s state and handle commands for that component.
 
-Which individual worker *actually has* write access is managed by SpatialOS, and can change regularly because of [load balancing](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#load-balancing). However, the list of workers that could *possibly* gain write access is constrained by the [access control lists](#access-control-list-acl).
+An entity’s [access control list (ACL)](#access-control-list-acl) governs which worker types can have write access (or authority) over each component.
 
-> Related:
+SpatialOS manages which specific worker instance has write access over a component. This can change regularly due to [load balancing](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#load-balancing).
+
+> **Related:**
 >
 > * [Understanding read and write access](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/design/understanding-access)
```

Comment:
link is broken, same for Read Access and authority entry

Reply:
probably one of these?

https://docs.improbable.io/reference/13.8/shared/authority-and-interest/introduction
https://docs.improbable.io/reference/13.8/shared/authority-and-interest/interest/active-read-access
https://docs.improbable.io/reference/13.8/shared/authority-and-interest/write-access-authority